src/libtiled/mapobject.h

```diff
@@ -99,16 +104,22 @@ class TILEDSHARED_EXPORT MapObject : public Object
      * Can be used to get/set property values using QVariant.
      */
     enum Property {
-        NameProperty,
-        TypeProperty,
-        VisibleProperty,
-        TextProperty,
-        TextFontProperty,
-        TextAlignmentProperty,
-        TextWordWrapProperty,
-        TextColorProperty
+        NameProperty             = 1 << 0,
+        TypeProperty             = 1 << 1,
+        VisibleProperty          = 1 << 2,
+        TextProperty             = 1 << 3,
+        TextFontProperty         = 1 << 4,
+        TextAlignmentProperty    = 1 << 5,
+        TextWordWrapProperty     = 1 << 6,
+        TextColorProperty        = 1 << 7,
+        CellProperty             = 1 << 8,
+        SizeProperty             = 1 << 9,
+        ShapeProperty            = 1 << 10,
+        RotationProperty         = 1 << 11
```

Comment:
Since you've added several entries here, I'm getting warnings in `MapObject::setMapObjectProperty` and `MapObject::mapObjectProperty`. I think size and rotation could simply be handled, but "shape" and "cell" are too special and it should probably catch those with a `Q_ASSERT(false);`.